sui/deploy-contract.js

```diff
@@ -8,48 +8,46 @@ const { loadSuiConfig, findPublishedObject } = require('./utils');
 
 // Add more contracts here to support more modules deployment
 const contractMap = {
-    gasService: {
+    GasService: {
         packageName: 'gas_service',
-        contractName: 'GasService',
-        displayName: 'Gas Service',
-        chainConfigKey: 'axelar_gas_service',
     },
 };
 
 async function processCommand(contractName, config, chain, options) {
     const contract = contractMap[contractName];
+    const packageName = options.packageName || contract.packageName;
```

Comment:
I think we should remove `contractMap` altogether. `Move.toml` for each of our packages contains
```
[package]
name = "AxelarGateway"
```
Could just recursively find the right package name from by searching (we do something similar to get the abi/bytecode for EVM builds by contract name, `getContractJSON` util).

You can add this in a separate PR though

Reply:
Ok, will do this separately.